pkg/nimble/Makefile

```diff
@@ -30,3 +30,7 @@ all: git-download
 	"$(MAKE)" -C $(TDIR)/contrib/
 
 include $(RIOTBASE)/pkg/pkg.mk
+
+ifeq (llvm,$(TOOLCHAIN))
```

Comment:
I am good with the change in general, I only would think that it makes sense to move this block up a bit in the file right below the other block disabling the warnings for nimble. Also a one line comment does not hurt?!

Reply:
Also works..

Reply:
Addressed and squashed